server/src/test/java/e2e/WorkflowEventsTest.java

```diff
@@ -1,31 +1,94 @@
 package e2e;
 
+import io.littlehorse.sdk.common.proto.AwaitWorkflowEventRequest;
+import io.littlehorse.sdk.common.proto.LHStatus;
+import io.littlehorse.sdk.common.proto.LittleHorseGrpc.LittleHorseBlockingStub;
+import io.littlehorse.sdk.common.proto.PutWorkflowEventDefRequest;
+import io.littlehorse.sdk.common.proto.RunWfRequest;
+import io.littlehorse.sdk.common.proto.VariableType;
+import io.littlehorse.sdk.common.proto.WfRunId;
+import io.littlehorse.sdk.common.proto.WorkflowEvent;
+import io.littlehorse.sdk.common.proto.WorkflowEventDefId;
+import io.littlehorse.sdk.wfsdk.WfRunVariable;
 import io.littlehorse.sdk.wfsdk.Workflow;
 import io.littlehorse.sdk.wfsdk.internal.WorkflowImpl;
 import io.littlehorse.test.LHTest;
 import io.littlehorse.test.LHWorkflow;
+import io.littlehorse.test.LHWorkflowEvent;
 import io.littlehorse.test.WorkflowVerifier;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 
 @LHTest
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class WorkflowEventsTest {
 
     @LHWorkflow("events")
     private Workflow eventsWf;
 
+    private LittleHorseBlockingStub client;
     private WorkflowVerifier verifier;
 
     @Test
-    public void shouldDoBasic() {
-        // Commented out until we add the ability to create a WorkflowEventDef to the test
-        // framework.
-        // verifier.prepareRun(eventsWf).waitForStatus(LHStatus.COMPLETED).start();
+    @Order(1)
+    void shouldBeAbleToGetWorkflowEventThrownFirst() {
+        verifier.prepareRun(eventsWf)
+                .waitForStatus(LHStatus.COMPLETED)
+                .thenVerifyWfRun(wfRun -> {
+                    WorkflowEvent result = client.withDeadlineAfter(1, TimeUnit.SECONDS)
```

Comment:
This is so bad. We should follow the same sitax. For example, `.thenWaitForWorkflowEvent(...)`

Reply:
Yeah...I thought you would say that, but I thought for the MVP of the test we could use this. Anyways, I can add the extra step but it's just a lot of work. Will do that in the next commit